mongo/filterstore.go

```diff
@@ -46,17 +47,18 @@ func NewMongoTimestamp(t time.Time, c uint32) (int64, error) {
 }
 
 // CreateFilterStore which can store, update and fetch filter jobs
-func CreateFilterStore(cfg config.MongoConfig, host string) (*FilterStore, error) {
+func CreateFilterStore(cfg config.MongoConfig, host string, shouldEnableMojorityWriteConcern, shouldEnableStrongReadConcern bool) (*FilterStore, error) {
```

Comment:
```suggestion
func CreateFilterStore(cfg config.MongoConfig, host string, enableMojorityWriteConcern, enableStrongReadConcern bool) (*FilterStore, error) {
```

Not sure the `should` adds any value to the name.